experiments/snippets/skuidCustom/skuidCustom.js

```diff
@@ -127,13 +76,23 @@ skuid.custom.fixCurrency = function (num) {
 	return parseFloat(num.toFixed(2));
 };
 
-// skuid.custom.clearArray(array)
-// force-clears the data out of an array which may still otherwise have data in it
-skuid.custom.clearArray = function (array) {
-	while (array.length) {
-		array.pop();
+// String.random(length)
+// Return a string of random (length) characters
+String.random = function (length) {
+	const radom13chars = function () {
+		return Math.random().toString(16).substring(2, 15)
 	}
-};
+	const loops = Math.ceil(length / 13)
+	return new Array(loops).fill(radom13chars).reduce((string, func) => {
+		return string + func()
+	}, '').substring(0, length)
+}
+
+// String.generateToken()
+// Generate a random 50 character token
+String.generateToken = function () {

```

Comment:
defining new functions on prebuilt globals like String and Date may cause issues when trying to use this code within Lightning Experience, generally we try to avoid modifying globals. Perhaps you could just assign these to new `skuid.custom` functions?

Reply:
Removed generateToken